neo-gui/Properties/Strings.Designer.cs

```diff
@@ -59,244 +59,289 @@ internal class Strings {
                 resourceCulture = value;
             }
         }
-
+        
         /// <summary>
-        ///   Looks up a localized string similar to About.
+        ///   查找类似 About 的本地化字符串。
         /// </summary>
         internal static string About {
             get {
                 return ResourceManager.GetString("About", resourceCulture);
             }
         }
-
+        
         /// <summary>
-        ///   Looks up a localized string similar to AntShares.
+        ///   查找类似 NEO 的本地化字符串。
```

Comment:
Could you preserve these comments in English?

Reply:
Sure, that is auto generated comments by VS. Thanks for suggestion.